lockfile/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "phylum_lockfile"
+description = "Package lock file parsers used by Phylum"
+version = "0.1.0"
+authors = ["Phylum, Inc. <engineering@phylum.io>"]
+edition = "2021"
+rust-version = "1.60"
+
+[dependencies]
+anyhow = "1.0.44"
+log = "^0.4.6"
+nom = "7.1.1"
+phylum_types = { git = "https://github.com/phylum-dev/phylum-types", branch = "development" }
+serde = { version = "^1.0", features = ["derive"] }
+serde_json = "^1.0"
+serde_yaml = "0.9.2"
+serde-xml-rs = "0.5.1"
+toml = "0.5.8"
```

Comment:
You might have just copied that over, but I would **always** specify the dependencies with major, minor AND patch version. Specifically for serde/serde_json here.

This makes sure that our semver compatability is actually correct. And rather than just blindly putting `.0` in the last position you should either test that `.0` actually works or use the latest version if you're just lazy (both are fine by me). This way when people see `.0` they **know** that we will actually build with `.0`.

Reply:
These lines with other version formats are copied verbatim from cli/cargo.toml. Should the cli crate dependencies also be updated?

Reply:
I'd appreciate it tbh. But I don't mind either way. I just wouldn't want it in **new** code. Feel free to update though if you know which versions are guaranteed to build.

Reply:
I'll just put in the versions from the package lock file since that's what CLI has been using.